cmake/Findjansson.cmake

```diff
@@ -55,9 +55,9 @@ if(JANSSON_FOUND)
   set(JANSSON_DEFINITIONS ${PC_JANSSON_CFLAGS_OTHER})
 endif()
 
-if(JANSSON_FOUND AND NOT TARGET Jansson::Jansson)
-  add_library(Jansson::Jansson UNKNOWN IMPORTED)
-  set_target_properties(Jansson::Jansson PROPERTIES
+if(JANSSON_FOUND AND NOT TARGET jansson::jansson)
+  add_library(jansson::jansson UNKNOWN IMPORTED)
+  set_target_properties(jansson::jansson PROPERTIES
```

Comment:
Fantastic! The casing can be annoying, but that seems to be the correct fix.

Reply:
Very. CMake could at least offer a "hint" message of some kind. "Target Jansson::Jansson not found (but there's a jansson::jansson, did you mean that?"